source/api_documentation/transactions_api/optional_fields.txt

```diff
@@ -387,32 +387,32 @@
   * - gender_data_append
     - Gender (Data Append)
     - Data from caller demographics lookup
-  
-  * - aku_list
+
+  * - sku_list
     - Sku List
-    - [Call center and Call Outcomes integrations] Total order amount (not including shipping) as reported by the call center or as a call outcome.
-
-  * - call_outcome_outcome_name		
-    - Outcome Name		
-    - The name describing the outcome event.		
-  
-  * - call_outcome_description		
-    - Call Outcome Description		
-    - Free form text for providing additional details about the outcome.		
-
-  * - call_outcome_partner_unique_id		
-    - Call Outcome Partner ID		
-    - Unique identifier, to distinguish between updating an existing outcome (for example correcting a sale that was reported) versus adding a second sale to the call (for example a reservation made while on the call and then an add on item purchased later).		
-
-  * - call_outcome_occurred_at		
-    - Call Outcome Occurred At		
-    - 10 digit time that the outcome occurred, in UTC seconds since 1/1/70, also known as Unix time_t.		
-
-  * - call_outcome_source		
-    - Call Outcome Source		
-    - The source of the call outcome.  Possible values are :UserOverride, :Api, :Import, :Expression, :Ivr, and :Machine		
-
-  * - call_outcome_value		
-    - Call Outcome Value		
-    - True or false as to whether the outcome was met or not.
+    - [Call center and Signal integrations] Total order amount (not including shipping) as reported by the call center or signal description.
+
+  * - signal_name
+    - Signal Name
+    - The name describing the signal event.
+
+  * - signal_description
+    - Signal Description
+    - Free form text for providing additional details about the signal.
+
+  * - signal_partner_unique_id
+    - Signal Partner ID
+    - Unique identifier, to distinguish between updating an existing signal (for example correcting a sale that was reported) versus adding a second sale to the call (for example a reservation made while on the call and then an add on item purchased later).
+
+  * - signal_occurred_at
+    - Signal Occurred At
+    - 10 digit time that the signal occurred, in UTC seconds since 1/1/70, also known as Unix time_t.
+
+  * - signal_source
+    - Signal Source
+    - The source of the signal.  Possible values are :UserOverride, :Api, :Import, :Expression, :Ivr, and :Machine
+
+  * - signal_value
+    - Signal Value
+    - True or false as to whether the signal was met or not.
```

Comment:
BTW is this field null for transactions that do note have a call_outcome attached to it? If so we should improve the docs with that note (similar to what we say in voice signal columns)